_posts/2016-08-28-elections.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Announcing the first VoxPupuli Elections
+date: 2016-08-28
+github_username: roidelapluie
+twitter_username: roidelapluie
+---
+
+Vox Pupuli is about to live its first elections. This blogpost will give you the
+context and how to become a candidate.
+
+> Vox Pupuli is a collective of Puppet module, tooling and documentation authors
+> all working together to ensure continued development on the code we maintain.
+
+(Source: [https://voxpupuli.org/](https://voxpupuli.org/))
+
+Vox Pupuli now represents 68 collaborators, and 86 repositories. In order to
```

Comment:
According to the website(which pulls from the github api) we have 93 Repositories.

I'd also say '68 member contributors'' and dozens more collaborators to include the folks that work with us but aren't members on the github org.